gno.land/pkg/integration/integration_test.go

```diff
@@ -1,11 +1,57 @@
 package integration
 
 import (
+	"strings"
 	"testing"
+
+	"github.com/stretchr/testify/assert"
```

Comment:
Yea, I'm on the other team's side 😛, but I'm still curious about others' opinions
while I agree that it can be a bit excessive for such comparison, the main reason to use `testify` here for me, is simply to maintain consistency across all tests in the codebase